src/util/ScillaUtil.ml

```diff
@@ -0,0 +1,5 @@
+module FilePathInfix = struct
```

Comment:
We already have a `Util.ml` in `src/lang`. Maybe that file should be moved here? or the contents of this file just go into that one?

Also, this file needs the usual copyright header.